packages/analytics-js/src/components/configManager/util/dataPlaneResolver.ts

```diff
@@ -29,10 +29,8 @@ const validateResidencyServerRegion = (
   residencyServerRegion?: ResidencyServerRegion,
   logger?: ILogger,
 ) => {
-  if (
-    residencyServerRegion &&
-    !Object.values(ResidencyServerRegion).includes(residencyServerRegion)
-  ) {
+  const residencyServerRegions = ['US', 'EU'];
+  if (residencyServerRegion && !residencyServerRegions.includes(residencyServerRegion)) {
```

Comment:
You can also remove the import for `ResidencyServerRegion`. Line 3

Reply:
It is required for line 29